.env

```diff
@@ -5,4 +5,5 @@ DATE_VERSION=20220328
 LATEST_DATE_VERSION=20220328
 MINIO_ADDRESS=minio:9000
 PULSAR_ADDRESS=pulsar://pulsar:6650
+#KAFKA_BROKER_LIST=localhost:9092
```

Comment:
I would suggest we don't support to setup with env.
All connection should be configured through milvus.yaml?